draft-ietf-rats-corim.md

```diff
@@ -996,6 +1000,26 @@ measurements for the Target Environment.
 {::include cddl/coswid-triple-record.cddl}
 ~~~
 
+#### Conditional Reference Series Triple {#sec-comid-triple-cond-ref}
+
+A Conditional Reference Series triple relates reference measurements to a Target
+Environment where endorsed measurements are accepted given all reference measurements
+are accepted. The triple subject is a compound statement consisting of a Target Environment
+and initial Reference Values that MUST be satisfied before evaluating the triple object.
+
+The triple object is a series of additional measurements that are evaluated in the order
+they appear in an array. The series has one or more records containing additional Reference
+Values that may contain Endorsement values. If Evidence matches the additional Reference
+Values, both the additional Reference Values and the Endorsed values, if any, are accepted.
```

Comment:
```suggestion
Values in a record, the Endorsed values in that record, if any, are accepted.
```

Reply:
The scope of the term 'record' could be ambiguous unless it is defined better. Existing text uses 'additional' to disambiguate the other values in the triple object (which could be regarded as a 'record'), but so could any of the subordinate statements within the triple object.

Reply:
This term 'record' is the one defined in preceding sentence "The series has one or more records containing ...".

Changing 'record' to 'member-record' could help eliminate ambiguity. I've updated my comments reflect the change.